CONTRIBUTING.md

```diff
@@ -193,11 +193,11 @@ As languages differ quite a lot, here is a set of captures available to you when
 @comment               ; line and block comments
 @comment.documentation ; comments documenting code
 
-@comment.error         ; error-type comments (e.g., `DEPRECATED:`)
-@comment.warning       ; warning-type comments (e.g., `WARNING:`, `FIX:`)
-@comment.hint          ; note-type comments (e.g., `NOTE:`)
+@comment.error         ; error-type comments (e.g. `DEPRECATED:`)
```

Comment:
```suggestion
@comment.error         ; error-type comments (e.g. `ERROR:`, `BUG:`)
```

Reply:
I'd like to keep "deprecated" in there -- somewhere -- as this is a _very_ common comment in Neovim ;)

Reply:
Then add it to the queries. ;)